### DISCUSSION TITLE
Add support for Multi-winner Ranked Choice Voting

### DISCUSSION THREAD

**Discussion by roargrrrr** [snapshot-labs/snapshot#2741] [Ideas]
Over at ManeNetDAO we are running a multiple winner election (aka electing a group of 5 individuals to seats) and realized RCV works for a single winner currently (I think?).  It would be awesome to have an implementation such as the one mentioned here: https://www.rcvresources.org/blog-post/multi-winner-rcv where we could select multiple winners.  Thanks!

**Comment by samwhosung** [comment]
We do not have plans to implement such a feature. But if you would like to give it a go yourself you can checkout this https://github.com/snapshot-labs/snapshot.js/tree/master/src/voting
